aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -178,17 +178,6 @@ bool check_sm_version(cudaDeviceProp * dprops) {
 }
 
 #if USE_ROCM
-c10::once_flag gcn_arch_override_flag;
-const char* over_arch = nullptr;
-
-void init_gcn_arch_override() {
-  over_arch = std::getenv("PYTORCH_DEBUG_FLASH_ATTENTION_GCN_ARCH_OVERRIDE");
-  if (over_arch) {
-      TORCH_WARN("SDPA functions only loads value from PYTORCH_DEBUG_FLASH_ATTENTION_GCN_ARCH_OVERRIDE once. "
-                 "Later changes to this environment variable with os.environ "
-                 "(or other methods) will not affect SDPA function's behavior.");
-  }
-}
 #endif
```

Comment:
This `#ifdef USE_ROCM` guard can be removed as well.

Reply:
Oh I forgot, those two variables, similar to a large potion of this file, is in an anonymous namespace.

Reply:
Good point, so perhaps this wasn't at all that necessary.